### ISSUE TITLE
Simplification throws error when encountering somevalue/novalue qualifiers

### ISSUE THREAD

**Issue by noinkling** [maxlath/wikidata-sdk#47]
Real examples: 
- https://www.wikidata.org/wiki/Special:EntityData/Q19180293.json?oldid=845659743
- https://www.wikidata.org/wiki/Special:EntityData/Q129809.json?oldid=778476685

```
var e = data.entities.Q19180293
wdk.simplify.qualifiers(e.claims.P1433[0].qualifiers.P156[0])
```
```
TypeError: Cannot destructure property `value` of 'undefined' or 'null'.
    at Object.simplifyClaim (/<...>/node_modules/wikidata-sdk/lib/helpers/simplify_claims.js:96:17)
    at parse (/<...>/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:84:31)
    at Object.entity (/<...>/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:9:40)
    at prefixedId (/<...>/node_modules/wikidata-sdk/lib/helpers/parse_claim.js:18:21)
```
Which means of course that following functions all crash too:
```
wdk.simplify.propertyQualifiers(e.claims.P1433[0].qualifiers.P156)
wdk.simplify.qualifier(e.claims.P1433[0].qualifiers)
wdk.simplify.claim(e.claims.P1433[0], { keepQualifiers: true })
wdk.simplify.propertyClaims(e.claims.P1433, { keepQualifiers: true })
...
wdk.simplify.entities(data.entities, { keepQualifiers: true })
```

Setting `somevalueValue` and `novalueValue` doesn't help.

P.S. The naming for the `simplify.qualifier` and `simplify.qualifiers` functions is a little confusing, as if they're swapped. But they are documented correctly. Was this intentional?

**Comment by maxlath** [comment]
I added support for `novalue` and `somevalue` qualifiers in e539d1f and fixed the `simplify.qualifier` and `simplify.qualifiers` swap in ed0e7a4.

Patches published in `v5.16.1`

Thank you for reporting those issues!

**Comment by noinkling** [comment]
Thank you 👍 

I think the naming swap should be considered a breaking change and probably requires a major version bump though, even if that was technically a mistake.
